standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/dataconnector/jdbc/AbstractJDBCConnectorProvider.java

```diff
@@ -192,6 +215,7 @@ protected Connection getConnection() {
       }
 
       table = buildTableFromColsList(tableName, cols);
+      table.setDbName(scoped_db);
```

Comment:
I think this might be wrong. 
So scoped_db is the name of the remote database thats being mapped. and table.setDbName() is meant to be the name of the hive database for this table. So for example,
MySQL has a database called "employees" with 2 tables (profiles and salaries).
Using a MySQL dataconnector, we map this remote db to a hive database called "companyx_employees" like so
create remote database companyx_employees using mysql_conn DBPROPERTIES("connector.remoteDbName"="employees");
show tables in this db should show the 2 tables "profiiles" and "salaries"
But to reference them in hive queries, you will have to use "companyx_employees.profiles" and NOT "employees.profiiles" as employees database in hive does not exist.

Reply:
Good catch. Will change!